python/upload_templates.py

```diff
@@ -247,6 +259,12 @@ def find_channels_with_max_peak_to_peak_vectorized(templates):
         templates_extension = analyzer.get_extension("templates")
         templates_extension_data = templates_extension.get_data(outputs="Templates")
 
+        # Do a check for the expected shape of the templates
+        number_of_units = sorting.get_num_units()
+        expected_shape = (number_of_units, 240, 384)
```

Comment:
```suggestion
        expected_shape = (number_of_units, target_nbefore + target_nafter, 384)
```
easier to read :)

Reply:
Yes, same for the number of channels right? Let me correct that.

Reply:
Done in the last commit. I also forgot to stage the files to delete templates : P